test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -235,6 +323,71 @@ class AssetListPage {
       `Expected number of token items ${expectedNumber} is displayed.`,
     );
   }
+
+  /**
+   * Checks if the token's general increase or decrease percentage is displayed correctly
+   *
+   * @param address - The token address to check
+   * @param expectedChange - The expected change percentage value (e.g. '+0.02%' or '-0.03%')
+   */
+  async check_tokenGeneralChangePercentage(
+    address: string,
+    expectedChange: string,
+  ): Promise<void> {
+    console.log(
+      `Checking token general change percentage for address ${address}`,
+    );
+    const isPresent = await this.driver.isElementPresentAndVisible({
+      css: this.tokenPercentage(address),
+      text: expectedChange,
+    });
+    if (!isPresent) {
+      throw new Error(
+        `Token general change percentage ${expectedChange} not found for address ${address}`,
+      );
+    }
+  }
+
+  /**
+   * Checks if the token's percentage change element does not exist
+   *
+   * @param address - The token address to check
+   */
+  async check_tokenGeneralChangePercentageNotPresent(
+    address: string,
+  ): Promise<void> {
+    console.log(
+      `Checking token general change percentage is not present for address ${address}`,
+    );
+    const isPresent = await this.driver.isElementPresent({
+      css: this.tokenPercentage(address),
+    });
+    if (isPresent) {
+      throw new Error(
+        `Token general change percentage element should not exist for address ${address}`,
+      );
+    }
+  }
+
+  /**
+   * Checks if the token's general increase or decrease value is displayed correctly
+   *
+   * @param expectedChangeValue - The expected change value (e.g. '+$50.00' or '-$30.00')
+   */
+  async check_tokenGeneralChangeValue(
+    expectedChangeValue: string,
+  ): Promise<void> {
+    console.log(`Checking token general change value ${expectedChangeValue}`);
+    const isPresent = await this.driver.isElementPresentAndVisible({
+      css: this.tokenIncreaseDecreaseValue,
+      text: expectedChangeValue,
+    });
+    if (!isPresent) {
+      throw new Error(
+        `Token general change value ${expectedChangeValue} not found`,
+      );
+    }
```

Comment:
nit: this simplifies a bit the code and readability. Note that waitForSelector already waits for an element with visible state by default

```suggestion
    await this.driver.waitForSelector({
      css: this.tokenIncreaseDecreaseValue,
      text: expectedChangeValue,
    });
```